R/write_eml.R

```diff
@@ -33,7 +33,7 @@
 #'   Data have been standardized to Darwin Core using the [movepub](
 #'   https://inbo.github.io/movepub/) R package and are downsampled to the first
 #'   GPS position per hour. The original data are available in Dijkstra et al.
-#'   (2023, <https://doi.org/10.5281/zenodo.10053903>), a deposit of Movebank
+#'   (2023, \doi{10.5281/zenodo.10053903}), a deposit of Movebank
```

Comment:
There's a bit of a conflict here between how Roxygen wants DOIs to be coded and how CrossRef wants DOIs to be displayed (full URLs). The result is here that the output on pkgdown:

<img width="706" height="154" alt="Image" src="https://github.com/user-attachments/assets/833f8d86-1e80-499d-aa02-3ff47617d4b3" />

looks a bit different from what is on the dataset page:

<img width="581" height="106" alt="Image" src="https://github.com/user-attachments/assets/87fdd412-df55-404b-83f8-cea7854cb5f9" />

I'm thinking we should ignore Roxygen here (but it might bite us again if we publish to CRAN).

Reply:
On second thought, not following Roxygen generates NOTES, so let's stick to Roxygen.